test/merge-tests.js

```diff
@@ -0,0 +1,82 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+const mergeProps = [
+  'deepProperties',
+  'configuration'
+];
+
+
+const build = (num) => {
+  const composable = {};
+
+  mergeProps.forEach(prop => {
+    composable[prop] = {
+      [num]: num,
+      merge: {
+        [num]: num
+      }
+    };
+  });
+
+  return { compose: composable };
```

Comment:
It would be a better idea to make `compose` a function like it is defined in the spec.

```js
Object.assign(() => {}, composable);
```